docs/operations.md

```diff
@@ -254,6 +254,49 @@ access.
 └──────────────────┘
 ```
 
+## Plugins
+
+wfx provides a flexible (out-of-tree) plugin mechanism to extend its HTTP request processing capabilities.
+A plugin operates as a subprocess initiated and supervised by wfx, thereby allowing plugins to be written in _any_
+programming language.
+
+### Enabling Plugins
+
+To enable plugins, wfx must be built with the `plugin` tag (enabled by default) and started with the
+`--client-plugins-dir`/`--mgmt-plugins-dir` flag.
+_Any executable_ in the respective directory is considered to be a plugin (symlinks to executables work as well).
+Plugins are sorted and executed in lexicographic order.
```

Comment:
[...] at wfx startup. 

We don't monitor this directory for plugins vanishing/showing up while running.

Reply:
True